source/_components/binary_sensor.iss.markdown

```diff
@@ -37,6 +37,6 @@ The [generic camera platform](/components/camera.mjpeg/) offers the possibility
 camera:
   - platform: generic
     name: ISS
-    still_image_url: {% raw %}https://maps.googleapis.com/maps/api/staticmap?center={{ states.binary_sensor.iss.attributes.lat }},{{ states.binary_sensor.iss.attributes.long }}&zoom=5&size=500x500&maptype=roadmap&markers=color:blue%7Clabel:P%7C{{ states.binary_sensor.iss.attributes.lat }},{{ states.binary_sensor.iss.attributes.long }}{% endraw %}
+    still_image_url: {% raw %}https://maps.googleapis.com/maps/api/staticmap?center={{ states.binary_sensor.iss.attributes.latitude }},{{ states.binary_sensor.iss.attributes.longitude }}&zoom=5&size=500x500&maptype=roadmap&markers=color:blue%7Clabel:P%7C{{ states.binary_sensor.iss.attributes.latitude }},{{ states.binary_sensor.iss.attributes.longitude }}{% endraw %}
     limit_refetch_to_url_change: true
```

Comment:
This change would only work if one is using `show_on_map: True` but not when running with the default. The idea was to enable the usage of ISS's location data but not show it on the map.